login-workflow/docs/services.md

```diff
@@ -50,6 +50,12 @@ constructor(pxbAuthConfig: PxbAuthConfig) {
 -   **customEmailValidator** (optional): _`ValidatorFn`_
     -   Custom email regex requirements.
     -   Default: `undefined`
+-   **customFirstNameRequirements** (optional): _`{ maxLength: number }`_
+    -   Custom first name requirements.
+    -   Default: `{ maxLength: 30 }`
+-   **customLastNameRequirements** (optional): _`{ maxLength: number }`_
+    -   Custom last name requirements.
+    -   Default: `{ maxLength: 30 }`
```

Comment:
If we want to match the implementation for R/RN, this lives as a single input (object) with keys for first and last name. That way we don't have to add a new input any time we add a new field. But I don't think we'll be adding a bunch of required fields anytime soon, so it's probably fine.

Reply:
Okay I'm cool with keeping it as is for now; I don't anticipate any additional first/last name customizations anytime soon.